README.md

```diff
@@ -1,2 +1,45 @@
-# fns-monorepo
-Solidity Backend &amp; Frontend DApp Monorepo
+# Flare Naming Service Monorepo
+
+## Introduction
+
+Flare Name Service (FNS) is a decentralized domain name registry that facilitates the allocation of human-readable addresses to wallet addresses and smart contracts. FNS domains serve to simplify the identification of wallets and smart contracts on the Flare blockchain, as well as to enable links to decentralized webpages.
+
+Anyone with a Flare-supported wallet may visit the FNS dApp to register a .flr address of their choosing. This process effectively transforms their 64-character 0x address, which is typically composed of randomized alphanumeric characters, into an easily recognizable domain name. The protocol also allows the setting of records of the users choosing, from arbitrary text inputs like email addresses, to wallet addresses for a multitude of other chains.
+
+Access the docs [here](https://docs.flrns.domains/).
+
+
+## Navigating the Monorepo
+
+- [Front End](./apps/next) - nextJS environment for building the [FNS dApp](https://app.flrns.domains/).
+
+- [Smart Contracts](./apps/forge) - Foundry environment for testing, compiling, and development of the smart contract protocol.
+
+
+## How it Works
+
+### FNS App
+
+To get started, navigate to the FNS app at app.flrns.domains using a Web3 compatible browser and a Flare browser extension wallet such as Metamask (see a list of compatible wallets here). You can utilize the search function to browse through domain name options and determine which domains are currently available. Additionally, the app allows you to view the details of previously registered domain names, including the address that holds ownership, the address to which they resolve, and the expiration date of their registration.
```

Comment:
Is there supposed to be a link to compatible wallets on the `here` word?

Reply:
Ah yes. Resolved